code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -15,6 +15,7 @@
 		var/obj/item/clothing/suit/space/hardsuit/C = wear_suit
 		J = C.jetpack
 	if(istype(J) && J.allow_thrust(0.01, src))
+		pass_flags |= PASSTABLE

```

Comment:
Put on jetpack
Turn on jetpack. PASSTABLE is not set because the gravity is currently on.
Gravity goes out. PASSTABLE is not set because that happens when the jetpack is turned on.